src/darwin/Framework/CHIPTests/CHIPSetupPayloadParserTests.m

```diff
@@ -190,7 +190,7 @@ - (void)testQRCodeParser_Error
 {
     NSError * error;
     CHIPQRCodeSetupPayloadParser * parser =
-        [[CHIPQRCodeSetupPayloadParser alloc] initWithBase38Representation:@"CH:J5L900CK70WWI0000"];
+        [[CHIPQRCodeSetupPayloadParser alloc] initWithBase38Representation:@"NT:R5L90MP500K64J00000"];
```

Comment:
Why the change from invalid code to invalid URL prefix here?  Should keep testing invalid code, I'd think....

Reply:
I wasn't sure what the test case was supposed to test. I can switch to a code which results in the "invalid argument" error though.

Reply:
Done, by the way, while trying to find a candidate for the test case I found that we silently pass some invalid QR codes, so fixed that and added more test cases.